src/views/users/User/UserCreateUpdate.vue

```diff
@@ -67,9 +67,7 @@ export default {
             if (formValue.update_password) {
               return true
             }
-
-            // 特权账号版本无需使用 Router 跳转因此不会携带 update 字段 this.$route.meta.action !== 'update'
-            return formValue.source !== 'local'
+            return formValue.source !== 'local' || this.$route.params.action !== 'update'
           }
         },
         password: {
```

Comment:
There are no significant code differences to notice in the given snippet of code. However, there is a suggestion that could potentially improve performance by combining the two functions into one using an arrow function:

```
export default {
  password: form => {
    if (formValue.update_password) {
      return true;
    }

    if (this.$route.meta.action === 'update') {
      return formValue.source !== 'local';
    } else {
      return this.form.isDirty && formValue.source !== 'local';
    }
  },
};
```